spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/ServerProperties.java

```diff
@@ -1181,6 +1218,113 @@ else if (handler instanceof HandlerCollection) {
 			});
 		}
 
+		public static class Accesslog {
+
+			/**
+			 * Enable access log.
+			 */
+			private Boolean enabled;
+
+			/**
+			 * Jetty accesslog filename.
+			 */
+			private String filename;
```

Comment:
Jetty is redundant here. Can you be a bit more explicit? What is the format? Does that contain the file extension or is it a name that Jetty will use to compute the file name?